src/common/BaseAnnotator.ts

```diff
@@ -37,6 +38,8 @@ export type Options = {
 export default class BaseAnnotator extends EventEmitter {
     container: Container;
 
+    initialized = false;
```

Comment:
We've been trying to avoid adding state to our annotators, when possible. Should we move `initialized` to the store, dispatch an action to update it, then emit an event using our middleware?